rcl/include/rcl/publisher.h

```diff
@@ -152,8 +152,7 @@ rcl_publisher_init(
   const rcl_node_t * node,
   const rosidl_message_type_support_t * type_support,
   const char * topic_name,
-  const rcl_publisher_options_t * options
-);
+  const rcl_publisher_options_t * options);
```

Comment:
There are other functions in this same file with a different style, mind changing them to have the same style as this (or the previous one before this change)?